### PR TITLE
Place additional setup config under collapse content

### PR DESCRIPTION
**Pull request by estherk15**
### What does this PR do? What is the motivation?

- Place additional beta configuration inside a collapse content
- Add additional beta configuration for Kafka
- [DOCS-7905](https://datadoghq.atlassian.net/browse/DOCS-7905)

### Merge instructions

- [ ] Please merge after reviewing

### Additional notes

[DOCS-7905]: https://datadoghq.atlassian.net/browse/DOCS-7905?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/en/universal_service_monitoring/setup.md [hunk 1]
```diff
@@ -762,16 +765,19 @@ DD_SYSTEM_PROBE_PROCESS_SERVICE_INFERENCE_ENABLED=true
...
 Universal Service Monitoring is available in <strong>beta</strong> to monitor TLS encrypted traffic from services implemented in Golang.
 </div>
 
-**Note**:
-* Go HTTPS servers can upgrade HTTP1.1 protocol to HTTP/2 which is supported in private beta. Reach to your account manager for further details
-* Requires Agent version 7.51 or greater.
+<strong>Note</strong>:
+<br>
+<ul role="list">
+  <li>Go HTTPS servers can upgrade HTTP1.1 protocol to HTTP/2 which is supported in private beta. Reach to your account manager for further details</li>
```
Comment (jhgilbert) [inline content/en/universal_service_monitoring/setup.md:778]:
```suggestion
  <li>Go HTTPS servers can upgrade HTTP1.1 protocol to HTTP/2 which is supported in private beta. Reach out to your account manager for details.</li>
```

File: content/en/universal_service_monitoring/setup.md [hunk 2]
```diff
@@ -931,6 +937,63 @@ agents:
...
+  serviceMonitoring:
+    enabled: true
+
+agents:
+  ...
+  containers:
+    systemProbe:
+      env:
+        - name: DD_SERVICE_MONITORING_CONFIG_ENABLE_KAFKA_MONITORING
+          value: " true"
```
Comment (jhgilbert) [inline content/en/universal_service_monitoring/setup.md:990]: This seems like a strange value, should it just be `true`? Or have the space removed?

**Review by jhgilbert — APPROVED** [review]
Approved with minor suggestions and a query around one of the values, which looks strange to me. LMK if you'd like another look at any point. Thank you!

File: content/en/universal_service_monitoring/setup.md (hunk 2 above)
Comment (estherk15) [reply content/en/universal_service_monitoring/setup.md:990]: @conward2121 Is this value correct?

Comment (estherk15) [reply content/en/universal_service_monitoring/setup.md:990]:
```suggestion
          value: "true"
```

**Pull request merged**